src/woo_search/search_index/tests/test_search_api.py

```diff
@@ -380,6 +380,43 @@ def test_query_field_boosts(self):
             data["results"][4]["record"]["uuid"], "d21c2a2f-ad02-41d5-8754-d24ba7092090"
         )
 
+    def test_query_default_search_uses_AND_instead_of_OR(self):
+        index_publication(
+            **IndexPublicationFactory.build(
+                uuid="9c3360b8-2ce7-4742-9051-e586b686fc48",
+            )
+        )
+        index_document(
+            **IndexDocumentFactory.build(
+                uuid="37eb1144-a3da-48d1-b2fb-88075f781611",
+                publicatie="9c3360b8-2ce7-4742-9051-e586b686fc48",
+                identifier="Document one of many",
+            )
+        )
+        index_document(
+            **IndexDocumentFactory.build(
+                uuid="da45268a-ab21-4a81-bfc4-b0430edf339b",
+                publicatie="9c3360b8-2ce7-4742-9051-e586b686fc48",
+                identifier="Document two of many",
+            )
+        )
+
+        response = self.client.post(self.url, {"query": "document two"})
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        data = response.json()
+
+        self.assertEqual(data["count"], 1)
+        self.assertFalse(data["previous"])
+        self.assertFalse(data["next"])
+        # test if results have the same length as the count
+        self.assertEqual(len(data["results"]), 1)
```

Comment:
these assertions can be omitted as they have nothing to do with what is being tested - the assertion on the count is sufficient.